content/en/docs/tasks/administer-cluster/setup-ha-etcd-with-kubeadm.md

```diff
@@ -33,9 +34,37 @@ when using kubeadm to set up a kubernetes cluster.
 ## Setting up the cluster
 
 The general approach is to generate all certs on one node and only distribute
-the *necessary* files to the other nodes. Note that kubeadm contains all the
-necessary crytographic machinery to generate the certificates described below;
-no other cryptographic tooling is required for this example.
+the *necessary* files to the other nodes.
+
+{{< note >}}
+**Note:**  kubeadm contains all the necessary crytographic machinery to generate
+the certificates described below; no other cryptographic tooling is required for
+this example.
+{{< /note >}}
+
+
+1. Configure the kubelet to be a service manager for etcd.
+
+    You must replace the default systemd unit file with a simpler configuration.
+    If you skip this step the kubelet will be crash-looping and the following
+    instructions will not work.
+
+    ```sh
+    rm -rf /etc/systemd/system/kubelet.service.d
+    cat << EOF > /etc/systemd/system/kubelet.service
+    [Unit]
+    Description=The Standalone Kubelet
+
+    [Service]
+    ExecStart=/usr/bin/kubelet --pod-manifest-path=/etc/kubernetes/manifests --allow-privileged=true
```

Comment:
Why not just override the ExecStart of the drop-in with a higher precedence, say `/etc/systemd/system/kubelet.service.d/20-etcd-service-manager`:

```
[Service]
ExecStart=
ExecStart=/usr/bin/kubelet --pod-manifest-path=/etc/kubernetes/manifests --allow-privileged=true
```

Reply:
I've also had luck with just running the `kubeadm alpha phase certs ca` and `kubeadm alpha phase kubeconfig kubelet` commands with the default systemd unit config, but it does create a lot of noise with errors connecting to the api server, I'm not sure if this method resolves that.

Reply:
hmmm

I end up with this when testing that out. Reading more about systemd to see if there is another way to do this

Jun 26 18:52:23 ip-172-31-13-12 systemd[1]: kubelet.service: Service has more than one ExecStart= setting, which is only allowed for Type=oneshot services. Refusing.

Reply:
oh right, i have to clear it first

```
[Service]
ExecStart=
ExecStart=/usr/bin/kubelet --pod-manifest-path=/etc/kubernetes/manifests --allow-privileged=true
```
haha fixed! This is much better, thank you!